src/utils/tokens/binance-testnet-tokens.json

```diff
@@ -0,0 +1,10 @@
+[
+  {
+    "chain": "binance",
+    "address": "0x0000000000000000000000000000000000000000",
+    "name": "BNB",
+    "symbol": "BNB",
+    "decimals": 18,
+    "iconUrl": "https://images.prismic.io/pillar-app/d8e73973-82f1-4b20-a068-fe20e21816b0_0xbb4cdb9cbd36b01bd1cbaebf2de08d9173bc095c_1.png?auto=compress,format"
+  }
+]
```

Comment:
Remember new line here

Reply:
means?

Reply:
![Screenshot_20221207-075618](https://user-images.githubusercontent.com/1009271/206120774-bb7d050d-fbf5-450b-9389-c3b2340ae8ca.png)

See screenshot

Reply:
Added a new line in last